### PR TITLE
Add CUAV V5+ Build Log

### PR DESCRIPTION
**Pull request by Junkim3DR**
Full build log of CUAV V5+ with DJI F450 frame.

### CODE REVIEW

File: Add CUAV V5+ Build Log [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+The following tools are used in this assembly:
+
+- 2.0mm Hex screwdriver
+- 3mm Phillips screwdriver
+- Wire cutters
+- Precision tweezers
+
+## Assembly
+
+Estimated time to assemble frame is 2 hours and 1.5 hours installing the autopilot and configuring the airframe in *QGroundControl*.
```
Comment (Tony3dr) [inline Add CUAV V5+ Build Log]: - Estimated time to assemble frame is 2 hours and 1.5 hours installing the autopilot and configuring the airframe in *QGroundControl*.
+ Estimated time to assemble is 90 minutes, the frame is about 45 minutes and 45 munites installing the autopilot and configuring the airframe in *QGroundControl*.

**Comment by hamishwillee** [comment]
@Junkim3DR @Tony3dr I'm going to close this PR shortly and recreate as a branch on this repo. This will allow me to edit the  content before submitting. I will also include the suggested fix in https://github.com/PX4/px4_user_guide/pull/523#pullrequestreview-262749881

**Comment by hamishwillee** [comment]
Note, this PR as is would be unworkable because the file is not even embedded in the gitbook structure or named as a markdown file. Perhaps next time copy another buildlog and follow the patterns?

**Comment by hamishwillee** [comment]
Replaced by #525

**Pull request closed without merging**